external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/CachedKafkaProducer.scala

```diff
@@ -37,9 +37,8 @@ private[kafka010] object CachedKafkaProducer extends Logging {
   private val defaultCacheExpireTimeout = TimeUnit.MINUTES.toMillis(10)
 
   private lazy val cacheExpireTimeout: Long =
-    Option(SparkEnv.get).map(_.conf.getTimeAsMs(
-      "spark.kafka.producer.cache.timeout",
-      s"${defaultCacheExpireTimeout}ms")).getOrElse(defaultCacheExpireTimeout)
+    Option(SparkEnv.get).map(_.conf.get(PRODUCER_CACHE_TIMEOUT)
+    ).getOrElse(defaultCacheExpireTimeout)
```

Comment:
indentation is wrong

Reply:
![1](https://user-images.githubusercontent.com/18002496/55472410-9dfbd980-563e-11e9-8342-5cb630c764a8.jpg)
?

Reply:
What I would do in this case is:

```scala
 private lazy val cacheExpireTimeout: Long = Option(SparkEnv.get)
    .map(_.conf.get(PRODUCER_CACHE_TIMEOUT))
    .getOrElse(defaultCacheExpireTimeout)
```
 
Or simply write the whole expression into one line by renaming `defaultCacheExpireTimeout` to something shorter.